database/src/migrations/20230719172000_project_type_clean_up.ts

```diff
@@ -0,0 +1,42 @@
+import { Knex } from 'knex';
+
+/**
+ * // add new table for region lookup and connection table to project and survey
+ *
+ * @export
+ * @param {Knex} knex
+ * @return {*}  {Promise<void>}
+ */
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`
+    set search_path= biohub, public;
+    
+    -- Get existing types and insert them into programs in order
+    INSERT INTO "program" (name, record_effective_date, description, record_end_date)
+    SELECT name, record_effective_date, description, record_end_date
+    FROM project_type
+    ORDER BY project_type_id;
```

Comment:
Any reason for the order by? I assume it wouldn't make a difference either way, overall?